src/nginxconfig/generators/to_conf.js

```diff
@@ -48,7 +48,7 @@ const longestKey = items => {
     return longest;
 };
 
-const recurse = (entriesOrObject, depth) => {
+const recurse = (entriesOrObject, depth, endLine = ';') => {
```

Comment:
```suggestion
const recurse = (entriesOrObject, depth) => {
```
This change can be reverted